core/arch/arm/plat-rzg/link.mk

```diff
@@ -0,0 +1,14 @@
+include core/arch/arm/kernel/link.mk
+
+all: $(link-out-dir)/tee.srec
+cleanfiles += $(link-out-dir)/tee.srec
+
+cleanfiles += $(link-out-dir)/tee-raw.bin
+$(link-out-dir)/tee-raw.bin: $(link-out-dir)/tee.elf scripts/gen_tee_bin.py
+	@$(cmd-echo-silent) '  GEN     $@'
+	$(q)scripts/gen_tee_bin.py --input $< --out_tee_raw_bin $@
+
+cleanfiles += $(link-out-dir)/tee.srec
+$(link-out-dir)/tee.srec: $(link-out-dir)/tee-raw.bin
+	@$(cmd-echo-silent) '  SREC    $@'
+	$(q)$(OBJCOPYcore) -I binary -O srec $< $@
```

Comment:
I propose moving the `tee-raw.bin` and `tee.srec` recipes to `core/arch/arm/kernel/link.mk` since they are also used by `plat-rcar`. You would keep only this here and in `plat-rcar/link.mk`:
```make
all: $(link-out-dir)/tee.srec
cleanfiles += $(link-out-dir)/tee.srec $(link-out-dir)/tee-raw.bin
```

Reply:
Agreed.

Reply:
Just the below should be suffice in platform link file,  

`all: $(link-out-dir)/tee.srec`

Reply:
Yes, although in that case you will need to add the files unconditionally to `cleanfiles`. I'm fine with that.